pymeasure/instruments/yokogawa/aq6370d.py

```diff
@@ -0,0 +1,208 @@
+        ":SENSe:SWEep:POINts %d",
+        "Control the sample number (int from 51 to 50001).",
+        validator=strict_range,
+        values=[51, 50001],
+        get_process=lambda x: int(x),
+    )
+
+    # Wavelength settings (all assuming wavelength mode, not frequency mode) -----------------------
+
+    wavelength_center = Instrument.control(
+        ":SENSe:WAVelength:CENTer?",
+        ":SENSe:WAVelength:CENTer %g",
+        "Control measurement condition center wavelength (float in m).",
+        validator=strict_range,
+        values=[50e-9, 2250e-9],
+    )
+
+    wavelength_span = Instrument.control(
+        ":SENSe:WAVelength:SPAN?",
+        ":SENSe:WAVelength:SPAN %g",
+        "Control wavelength span (float from 0 to 1100e-9 m).",
+        validator=strict_range,
+        values=[0, 1100e-9],
+    )
+
+    wavelength_start = Instrument.control(
+        ":SENSe:WAVelength:STARt?",
+        ":SENSe:WAVelength:STARt %g",
+        "Control the measurement start wavelength (float from 50e-9 to 2250e-9 in m).",
+        validator=strict_range,
+        values=[50e-9, 2250e-9],
+    )
+
+    wavelength_stop = Instrument.control(
+        ":SENSe:WAVelength:STOP?",
+        ":SENSe:WAVelength:STOP %g",
+        "Control the measurement stop wavelength (float from 50e-9 to 2250e-9 in m).",
+        validator=strict_range,
+        values=[50e-9, 2250e-9],
+    )
+
+    # Trace operations -----------------------------------------------------------------------------
+
+    active_trace = Instrument.control(
+        ":TRACe:ACTive?",
+        ":TRACe:ACTive %d",
+        "Control the active trace (str 'TRA', 'TRB', 'TRC', ...).",
+    )
+
+    def copy_trace(self, source, destination):
+        """
+        Copy the data of specified trace to the another trace.
+
+        :param source: Source trace (str 'TRA', 'TRB', 'TRC', ...).
+        :param destination: Destination trace (str 'TRA', 'TRB', 'TRC', ...).
+        """
+        self.write(f":TRACe:COPY {source},{destination}")
+
+    def delete_trace(self, trace):
+        """
+        Delete the specified trace.
+
+        :param trace: Trace to be deleted (str 'ALL', 'TRA', 'TRB', 'TRC', ...).
+        """
+        if trace == "ALL":
+            self.write(":TRACe:DELete:ALL")
+        else:
+            self.write(f":TRACe:DELete {trace}")
+
+    def get_xdata(self, trace="TRA"):
+        """
+        Measure the x-axis data of specified trace, output wavelength in m.
+
+        :param trace: Trace to measure (str 'TRA', 'TRB', 'TRC', ...).
+        :return: The x-axis data of specified trace.
+        """
+        return self.values(f":TRACe:X? {trace}")
+
+    def get_ydata(self, trace="TRA"):
+        """
+        Measure the y-axis data of specified trace, output power in dBm.
+
+        :param trace: Trace to measure (str 'TRA', 'TRB', 'TRC', ...).
+        :return: The y-axis data of specified trace.
+        """
+        return self.values(f":TRACe:Y? {trace}")
```

Comment:
As already discussed, we can now use either "A" or "TRA". Regarding using channels, I don't think we'll be switching much, but it might be a good edit for the future!